charts/nebula-cluster/values.yaml

```diff
@@ -1,6 +1,6 @@
 nebula:
   version: v3.4.0
-  imagePullPolicy: Always
```

Comment:
You can update the policy during installation, but I do not recommend changing the default values.